kahuna/public/js/search/results.html

```diff
@@ -27,8 +27,24 @@
 
             <div class="results-toolbar-item results-toolbar-item--static results-toolbar-item--left
                         results__related-labels related-labels flex-container text-small"
-                 ng:if="ctrl.relatedLabels.length !== 0">
-                <div class="related-labels__title">Related to {{ctrl.parentLabel}}:</div>
+                        ng:class="{'related-labels--with-labels': ctrl.relatedLabels.length !== 0}">
+                <div class="related-labels__title">
+                    Related to:
+                    <span ng:switch="ctrl.relatedLabels.length === 0">
+                        <form class="inline-block" ng:switch-when="true" ng:submit="ctrl.setParentLabel()">
+                            <gr-datalist class="related-labels__datalist" gr:search="ctrl.labelSearch(q)">
+                                <!-- TODO: autogrow this element -->
+                                <input class="related-labels__parent" type="text"
+                                       placeholder="e.g. culture"
```

Comment:
I think case insensitive as that's what others have been doing so far.
I'll change this to the "observer", but we could populate the example from the list later on.